### PR TITLE
Update config docs for the new color object syntax

### PR DESCRIPTION
**Pull request by mike-engel**
This documents the change from an array to an object for the colors. Directly related to zeit/hyperterm#193

### CODE REVIEW

**Comment by rauchg** [comment]
We should probably explain where the keys come from, and link to the complete list?

**Comment by mike-engel** [comment]
@rauchg: Added a link to the default config. Not sure if you were looking for something more, or less than that.

File: index.html [hunk 1]
```diff
@@ -533,8 +533,8 @@ <h2 id="config"><a href="#"config">Configuration</a></h2>
             </tr>
             <tr>
               <td>"colors"</td>
-              <td>["#000000", "#ff0000", ...]</td>
-              <td>A list of overrides for the color palette</td>
+              <td>{ black: "#000000", red: "#ff0000", ... }</td>
+              <td>A list of overrides for the color palette. The names of the keys represent the "ANSI 16", which can all be seen <a href="https://github.com/zeit/hyperterm/blob/master/config-default.js#L32">in the default config</a>.</td>
```
Comment (rauchg) [inline index.html]: This links is a moving target, since it points to `master`. We'll have to update it to a specific commit.

Comment (mike-engel) [inline index.html]: Good point. Updated it to the commit in the PR from hyperterm. I'm not planning on rebasing, so it should be fine. If I rebase that PR, I'll update the link here.

**Comment by rauchg** [comment]
superb

**Pull request merged**